src/scss/components/_button.scss

```diff
@@ -222,4 +198,32 @@ $button-padding-h: $grid-margin-2;
       color: var(--of-button-danger-bg);
     }
   }
+
+  // anchor variants, these map to utrecht components:
+  // * UtrechtButton (the default)
+  // * UtrechtButtonLink (`a` component)
+  // * UtrechtLinkButton (`button` component)
+  @include bem.modifier('anchor') {
+    @include button-theme(
+      '',
+      var(--of-button-anchor-fg, var(--utrecht-link-color)),
+      var(--of-button-anchor-bg),
+      var(--of-button-anchor-color-border),
+      $color-background-hover: var(--of-button-anchor-hover-bg, var(--utrecht-link-hover-color)),
+      $color-border-hover: var(--of-button-anchor-hover-color-border),
+      $color-active: var(--of-button-anchor-active-fg),
+      $color-background-active: var(--of-button-anchor-active-bg),
+      $color-border-active: var(--of-button-anchor-active-color-border),
+      $color-border-focus:
+        var(--of-button-anchor-focus-color-border, var(--utrecht-button-focus-border-color)),
+      $force-colors: true // there will not be an anchor variant
+    );
+
+    // the OPPOSITE of what we normally do
+    text-decoration: var(--utrecht-link-hover-text-decoration, none);
+    &:hover {
+      --of-button-anchor-fg: var(--utrecht-link-hover-color);
+      text-decoration: var(--utrecht-link-text-decoration, underline);
+    }
+  }
```

Comment:
Maybe I am doing something wrong, but at the moment when I hover on the "previous page" button, i don't get the right hover colour, I get `--_utrecht-button-hover-color`

Reply:
I'll re-check this one! The buttons is definitely the most nasty part.

Reply:
in the SDK this seems to work for me when I specify the `--utrecht-link-hover-color: fuchsia;` design token :thinking:

Reply:
checked via video call and this seems to be okay